ui/src/routes/+page.server.ts

```diff
@@ -21,19 +21,34 @@ const limiter = new RateLimiter({
   }
 });
 
+const publicServiceAnnouncement = "Public Service Announcement: Ask Astro is currently undergoing maintenance and will be back shortly. We apologize for any inconvenience this may cause!";
```

Comment:
I saw there's a similar string in `api/ask_astro/services/questions.py`. Is it something we should reuse?

Reply:
It's the exact same string with the difference being this has `Public Service Announcement` in front. I don't think you can reuse the varaible as one is from our `api` folder which is backend, the other is in our `ui` folder which is frontend. They are 2 different services.

The reason why they in the backend is because even though we can display the message on the UI, but someone can get around the UI and still send a request to ask astro (e.g. through slack or somehow hit our backend api directly)